### ISSUE TITLE
Notes does no longer follow the formatting in org-ref-note-title-format

### ISSUE THREAD

**Issue by Horsevad** [jkitchin/org-ref#729]
Both when I create a new bibliography entry with org-ref-pdf-to-bibtex  or when in the .bib file uses org-ref-open-bibtex-notes,  the newly created note heading in the org file notes is created wrongly

From this bibtex entry:

@article{dahl09_does_movie_violen_increas_violen_crime,
  author =	 {Gordon Dahl and Stefano DellaVigna},
  title =	 {Does Movie Violence Increase Violent Crime?*},
  journal =	 {Quarterly Journal of Economics},
  volume =	 124,
  number =	 2,
  pages =	 {677-734},
  year =	 2009,
  doi =		 {10.1162/qjec.2009.124.2.677},
  url =		 {https://doi.org/10.1162/qjec.2009.124.2.677},
  DATE_ADDED =	 {Wed Apr 22 02:19:39 2020},
}

with org-ref-open-bibtex-notes  it opens the  .org file, with an interactive window with : 

* Dahl, G., & DellaVigna, S. (2009): Does movie violence increase violent crime?*
  :PROPERTIES:
  :Custom_ID: dahl09_does_movie_violen_increas_violen_crime
  :END:

Where it can be saved or refiled. 

my org-ref-note-title-format is set as:
(setq org-ref-note-title-format
  (concat "* MAYBE %y - %t
 :PROPERTIES:
  :INTERLEAVE_PDF: ~/archive/bibtex-pdfs/%k.pdf
  :Custom_ID: %k
  :AUTHOR: %a
  :JOURNAL: %j
  :YEAR: %y
  :VOLUME: %v
  :PAGES: %p
  :DOI: %D
  :URL: %U
:END:
"))

**Comment by HaoZeke** [comment]
I seem to have run into this issue as well. How was it resolved? @Horsevad

**Comment by Horsevad** [comment]
There used to be some inconsistency with how notes were created, for me I noticed between the notes created by org-ref-pdf-to-bibtex in a pdf, and org-ref-open-bibtex-notes. Before I has this fixed by changing the function org-ref-notes-function in my .emeacs file, to have the same behavior.

However the way notes are created / opend was changed recently in: 
https://github.com/jkitchin/org-ref/commit/875371a63430544e446db7a76b44b33c7e20a8bd

So now I get the desired behavior without changes to the org-ref-notes-function.

I hope this was helpfull,  the original solution I think I found in some email list or an old blog post so it might not be applicable to you.
